charts/turing/templates/turing-service-account.yaml

```diff
@@ -0,0 +1,8 @@
+{{- if .Values.clusterConfig.useInClusterConfig -}}
```

Comment:
We should be able to:
- have a service account not only for in-cluster deployments
- set annotations for the service account created
- use a provided service account

You can check https://github.com/caraml-dev/helm-charts/blob/main/charts/mlp/templates/service-account.yaml for reference

Reply:
Updated.